File: ABC/ABC337/A/main.cc
#include <iostream>

using namespace std;

int main() {
  int n;
  cin >> n;
  int st = 0, sa = 0;
  while (n--) {
    int x, y;
    cin >> x >> y;
    st += x;
    sa += y;
  }

  if (st > sa) {
    cout << "Takahashi\n";
  } else if (st < sa) {
    cout << "Aoki\n";
  } else {
    cout << "Draw\n";
  }

  return 0;
}

File: ABC/ABC337/A/main.py
n = int(input())
st = sa = 0
for _ in range(n):
    x, y = map(int, input().split())
    st += x
    sa += y

res = 'Draw'
if st != sa:
    res = 'Takahashi' if st > sa else 'Aoki'
print(res)